dist/image-sequencer-ui.js

```diff
@@ -725,8 +725,8 @@ function generatePreview(previewStepName, customValues, path, selector) {
 
     var previewSequencerSteps = {
       "resize": "125%",
-      "brightness": "20",
-      "saturation": "5",
```

Comment:
Why has this changed in the dist but not in the src js files?

Reply:
This must have come by mistake!

Reply:
And these changes come from here https://github.com/publiclab/image-sequencer/commit/30e9f359d5af33c0f36c2fe2d0cb6e042471a660 , the code merged didn't commit the dist file changes! so when I compiled it , the changes were shown in dist files, and that's how!